packages/model-viewer/src/features/controls.ts

```diff
@@ -593,7 +593,9 @@ export const ControlsMixin = <T extends Constructor<ModelViewerElementBase>>(
 
     [$syncCameraTarget](style: EvaluatedStyle<Vector3Intrinsics>) {
       const [x, y, z] = style;
-      this[$scene].setTarget(x, y, z);
+      if (!this[$renderer].arRenderer.isPresenting) {
+        this[$scene].setTarget(x, y, z);
```

Comment:
hmm, does this work? I think the `updateTarget()` call below does kind of the same thing. Also, I'm a little less sure about this bug now; what's wrong with the target moving the object around? In this case our state could be unsynced when we return from AR to 3D mode. Is it such a problem to disable your camera-target changes when in AR mode externally to MV? I could imagine situations where that motion could be useful, and I think it's consistent with how the turntable works.

Reply:
It has been confirmed to work on devices running WebXR.
WebXR and 3DView are synchronized in terms of animation state and camera coordinates. However, I don't think that the camera in AR (your device) and the camera in 3DView are the same thing.
Zooming in and moving the camera should be done by moving your own device in AR.

Reply:
Yeah, that's fair, but I might argue `camera-target` is just poorly named. It acts much more like the turntable (moving the model). Still, I see your point. I'm struggling to find a use case where shifting the model around programmatically during an AR session makes any sense (especially since we don't even expose the camera's location). 

Thanks!

Reply:
It's certainly not strange if you think of it as a turntable.
If you think about use cases, there are something that float in the air rather than AR that is placed on the ground.
For example, we can think of 3D maps that can be viewed in 3D using AR.

Thanks for the approval.